src/libraries/System.IO.Pipes/src/System/IO/Pipes/NamedPipeClientStream.cs

```diff
@@ -88,8 +89,15 @@ public NamedPipeClientStream(string serverName, string pipeName, PipeDirection d
             {
                 IsCurrentUserOnly = true;
             }
-
-            _normalizedPipePath = GetPipePath(serverName, pipeName);
```

Comment:
Wouldn't be better to change GetPipePath implementation instead?

Reply:
Yes, that is better. Changed like that.